<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="29610" started-at="2019-08-20T05:36:46Z" finished-at="2019-08-20T05:37:15Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="29610" started-at="2019-08-20T05:36:46Z" finished-at="2019-08-20T05:37:15Z">
      <class name="sample.ExcelConsolidation.App">
        <test-method status="PASS" signature="loadData()[pri:0, instance:sample.ExcelConsolidation.App@5622fdf]" name="loadData" is-config="true" duration-ms="15" started-at="2019-08-20T05:36:46Z" finished-at="2019-08-20T05:36:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadData -->
        <test-method status="PASS" signature="initializeDriver()[pri:0, instance:sample.ExcelConsolidation.App@5622fdf]" name="initializeDriver" is-config="true" duration-ms="5576" started-at="2019-08-20T05:36:46Z" finished-at="2019-08-20T05:36:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDriver -->
        <test-method status="FAIL" signature="enterData()[pri:0, instance:sample.ExcelConsolidation.App@5622fdf]" name="enterData" duration-ms="23998" started-at="2019-08-20T05:36:51Z" finished-at="2019-08-20T05:37:15Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Admin Config Loaded expected [Index1] but found [Index]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Admin Config Loaded expected [Index1] but found [Index]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at sample.ExcelConsolidation.adminConfig.adminConfigscreen(adminConfig.java:26)
at sample.ExcelConsolidation.App.enterData(App.java:143)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:483)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterData -->
      </class> <!-- sample.ExcelConsolidation.App -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
